src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -837,38 +837,40 @@ public async Task EnableDatabases()
             await ToggleDisableDatabases(disable: false);
         }
 
+        [RavenAction("/admin/databases/toggle-indexing", "POST", AuthorizationStatus.Operator)]
+        public async Task ToggleIndexing()
+        {
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                var raftRequestId = GetRaftRequestIdFromQuery();
+                var disable = GetBoolValueQueryString("disable") ?? false;
```

Comment:
The `/admin/databases/dynamic-node-distribution` has `enable`. Let's stick to one?

Reply:
I would prefer disable everywhere, we probably can change the other endpoint, it is not exposed in any client api

Reply:
We have `SetDatabaseDynamicDistributionOperation` which uses the `/admin/databases/dynamic-node-distribution` endpoint.
Changed to `enable`